cv19index/predict.py

```diff
@@ -354,33 +364,48 @@ def do_run(
     writes out predictions.
     """
     logger.info(f"Running {model_fpath} using {input_fpath} to {output_fpath}")
-    run_df = read_frame(input_fpath, schema_fpath, empty_ok=True)
+    run_df = read_frame(input_fpath, schema_fpath)
 
     if not run_df.empty:
         model = read_model(model_fpath)
-
-        out = run_model(run_df, model, **kwargs)
+        out = run_xgb_model(run_df, model, **kwargs)
         write_predictions(out, output_fpath)
     else:
         #: If there are no inputs to run predictions on, bypass
         #: executing the model and output an successful empty file.
         write_predictions(run_df, output_fpath)
 
 
-def main():
+def do_run_claims(args):
+    demo_df = read_demo(args.demo)
+    claim_df = read_claim(args.claim)
+
+    if args.model == 'xgboost':
+        input_df = preprocess_xgboost(claim_df, demo_df)
+        input_df = input_df.set_index('personId', drop=True)
+
+        model = read_model(args.path_to_model)
+        predictions = run_xgb_model(input_df, model)
+        write_xgb_predictions(predictions)
+    else:
+        logger.error(f"{args.model} is not a valid model type. valid models are xgboost.")
+
+
+def parser():
     logging.basicConfig(level=logging.INFO)
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("input")
-    parser.add_argument("output")
+    parser.add_argument("demo")
```

Comment:
demo is confusing.  Not obvious that it is short for demographics.  Call it person or demographics.

Reply:
Agreed, has been changed to demographics.